README.md

```diff
@@ -43,7 +43,24 @@ brew upgrade fzf-make
 paru -S fzf-make
 ```
 
+## NixOS / Nix (package manager)
+
+`fzf-make` can be run from the repository (latest version)
+```sh
+nix run github:kyu08/fzf-make
+```
+
+Or from the nixpkgs (channel >= 23.05)
+```sh
+nix run nixpkgs#fzf-make
+```
```

Comment:
This command only executes fzf-make once, and does not appear to be an install command.
I would like to see the install procedure written here. (`nix-env -iA nixpkgs.${package-name}` or `nix profile install nixpkgs#${package-name}` seem applicable)

Reply:
> This command only executes fzf-make once, and does not appear to be an install command.
I would like to see the install procedure written here. 

Yes, but `nix-env -iA` is depreciated, the proper to have it install is to edit the configuration.nix file and either add it to the user packages, or system ones. There is also the possibility to add it on user packages with home manager, or as a flake input using the url, ... and there is many other ways, which is why I'm not sure how to approach it.

I'm not really aware of the `nix profile` command too.

Also, it run the command once, but it put the flake evaluated derivation within the cache, which allows you to run it multiple times without rebuilding it. That the way i use it tbh, it's way easier.